Typstify/Typstify/ContentView.swift

```diff
@@ -351,6 +351,7 @@ struct Navigator: View {
                                     insertingPhotoItem?.getFilename(completionHandler: { result in
                                         switch result {
                                         case .success(let name):
+                                                let fullPath = (projectURL?.appendingPathComponent(name).path) ?? ""
```

Comment:
The file should be written to `dominantFolder` (the currently opened file's folder) instead of just the project root.